ansible/roles/test/tasks/pfc_wd/functional_test/timer_test.yml

```diff
@@ -0,0 +1,60 @@
+- name: Flush logs
+  shell: logrotate -f /etc/logrotate.conf
+
+- name: Generate PFC storm on fanout switch
+  action: apswitch template=pfc_storm.j2
+  args:
+    host: "{{peer_device}}"
+    login: "{{peer_login}}"
+  connection: switch
+
+- name: Wait for queue to recover from PFC storm
+  pause:
```

Comment:
Same as above comment. To make sure queue is recovered, we need to stop sending the PFC frames first.

Reply:
At this point we no longer send PFC frames